scss/typography/_helpers.scss

```diff
@@ -75,4 +108,15 @@ $stat-font-size: 2.5rem !default;
     margin-#{$global-left}: 0;
     list-style: none;
   }
+  
+  // Use to create an initial aka dropcap
+  .dropcaps{
+    line-height: 1.5; // Disabling default line height of an element
```

Comment:
So to expand on this & the result... this means that `@include dropcaps` or adding the `.dropcaps` class will *change* the line height when applied to tags like p or h1... ie my leading paragraph where I want to include a dropcaps will look _fundamentally different_ than my other paragraphs... 

And further, this is not accessible to change via a var or mixin parameter.

I'm not sure this is the right solution. What if we made this a parameter to the mixin and computed the :first-letter lineheight from it?